client/src/views/FlightsAll.vue

```diff
@@ -39,3 +28,22 @@ await initData(ApiService.getFlights, {
   },
 });
 </script>
+
+<template>
```

Comment:
Why are now all templates at the bottom?

Reply:
New best practice that totally makes sense because most of the time you're working on the script, not the template.